Photon/IoTlib/src/pressure.cpp

```diff
@@ -0,0 +1,187 @@
+/******************************************************************
+Pressure sensor
```

Comment:
Agree with you, these pressure files could be more general and accept different sensors.
I'll think about your words and try to come up with a better solution where the user could choose between the other sensors and the code would instantiate the correct object based on the user's choice.